compiler/rustc_hir/src/lang_items.rs

```diff
@@ -311,6 +311,7 @@ language_item_table! {
     Try,                     sym::Try,                 try_trait,                  Target::Trait;
 
     // Language items from AST lowering
+    SliceLen,                sym::slice_len_fn,        slice_len_fn,               Target::Method(MethodKind::Inherent);
```

Comment:
```suggestion
    SliceLen,                sym::slice_len_fn,        slice_len_fn,               Target::Method(MethodKind::Inherent);

    // Language items from AST lowering
```
I guess I wasn't clear enough. This was what I actually meant.